cli/user/README.md

```diff
@@ -94,7 +95,55 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+
+The Pravega User CLI can be accessed through the Kubernetes pod such as the Segment Store pod, or externally if exposed to Controller/Segment Store endpoints.
```

Comment:
This sentence is a bit unclear. I propose to rephrase it as: `The Pravega User CLI can be used from inside the Kubernetes cluster (including the Segment Store pod that has the CLI tool built-in), or externally if your Pravega cluster publicly exposes the Controller/Segment Store endpoints.`